migration-util/src/test/java/edu/unc/lib/dcr/migration/content/ACLTransformationHelpersTest.java

```diff
@@ -378,7 +407,7 @@ private void assertNoRolesAssignedforAuthenticated(Resource bxc5Resc) {
 
     private void assertHasEmbargo(String embargoEndDate, Resource bxc5Resc) {
         assertTrue("Resource " + bxc5Resc.getURI() + " did not have expected embargo with end date " + embargoEndDate,
-                bxc5Resc.hasLiteral(CdrAcl.embargoUntil, embargoEndDate));
```

Comment:
this isn't verifying that the right embargo end date is set anymore, could you add that back in?